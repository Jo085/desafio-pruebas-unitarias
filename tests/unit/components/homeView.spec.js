import { shallowMount } from '@vue/test-utils'
import HomeView from '@/views/HomeView.vue'

describe('Componente HomeView.vue', () => {
   test('Validación de match con el snapshot', () =>{
    // Selección del sujeto de prueba
     const wrapper = shallowMount(HomeView)
     // Aserción 
     expect(wrapper.html()).toMatchSnapshot()
   })
})