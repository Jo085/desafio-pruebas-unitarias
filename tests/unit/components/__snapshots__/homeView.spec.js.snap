// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente HomeView.vue Validaci√≥n de match con el snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <posts-view-stub></posts-view-stub>
</div>
`;
